ocp_resources/network_attachment_definition.py

```diff
@@ -230,7 +234,11 @@ def to_dict(self):
             if not self.topology:
                 raise ValueError("topology is required")
             spec_config = self.res["spec"]["config"]
-            spec_config["name"] = self.network_name
```

Comment:
`self.network_name` is required but not used anymore?

Reply:
It was so important I take care of it in the code and don't trust the user, so this is redundant :)

Reply:
I don't understand this answer.

Reply:
I removed it entirely. It is taken care of here:
spec_config["netAttachDefName"] = f"{self.namespace}/{self.name}"